src/Kunstmaan/AdminBundle/Resources/ui/js/_pagepart-editor.js

```diff
@@ -300,11 +302,18 @@ kunstmaanbundles.pagepartEditor = function (window) {
                 for (var i = 0; i < modules.length; i++) {
                     // Check if there really is a module with the given name and it if has a public reInit function
                     if (typeof kunstmaanbundles[modules[i]] === 'object' && typeof kunstmaanbundles[modules[i]].reInit === 'function') {
-                        kunstmaanbundles[modules[i]].reInit();
+                        // prevent duplicate modules
+                        if ($.inArray(modules[i], uniqueModules) === -1) {
+                            uniqueModules.push(modules[i]);
+                        }
                     }
                 }
             }
         });
+
+        for (var i = 0; i < uniqueModules.length; i++) {
+            kunstmaanbundles[uniqueModules[i]].reInit();
```

Comment:
No real need for an extra for loop here. You could also reInit the modules in the first for loop.